leakcanary-analyzer/src/main/java/com/squareup/leakcanary/ShortestPathFinder.java

```diff
@@ -176,6 +176,13 @@ private void visitRootObj(LeakNode node) {
     RootObj rootObj = (RootObj) node.instance;
     Instance child = rootObj.getReferredInstance();
 
+    //We should ignore leaks where an android.os.Binder is the root of the leak.
+    //When you bind and unbind from a Service, the OS will tend to keep a reference to the Binder
+    // after we'd expect it to be collected.
+    if (HahaHelper.extendsFrom(child, "android.os.Binder")) {
```

Comment:
This works, but it might also ignore some real leaks: for example, the binder and service are still active and are perfectly OK, but they are holding onto an activity (booo) that's just been destroyed.

How about instead adding a new method to `ExcludedRef` to support ignoring subclasses? See `com.squareup.leakcanary.ExcludedRefs.Builder#clazz(java.lang.String, boolean)`. That way, we would still move forward with the analysis, but we would report it as ignorable, and you can ignore those or actually look at them.